src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
```

Comment:
According to the checklist, it's recommended to use constants for magic numbers. In this case, the number of balls (3) should be extracted into a constant with a meaningful name.